src/AmalgamatorSession.ts

```diff
@@ -106,8 +106,12 @@ export class AmalgamatorSession extends LoggingDebugSession {
 
     /* child processes XXX: A type that represents the union of the following datastructures? */
     protected childDaps: AmalgamatorClient[] = [];
+    protected addressMap: Map<number, [string, string, string]> = new Map<
```

Comment:
Please add some documentation to explain what is in this map - a structured value (e.g. an interface) would make it self documenting and that would be preferred.

This may be irrelevant once other issues are resolved.

Reply:
I have updated it.